ballerina/tests/resources/expected_results/input_variables_with_empty_input_object_value.json

```
@@ -0,0 +1,13 @@
+{
+    "errors": [
+        {
+            "message": "Variable \"$userName\" got invalid value {};Expected type String!. String cannot represent value: {}",
```

Shouldn't we have a space here?

```suggestion
            "message": "Variable \"$userName\" got invalid value {}; Expected type String!. String cannot represent value: {}",
```